### PR TITLE
add libneon27-gnutls-dev rosdep key

### PR DESCRIPTION
**Pull request by pietrocolombo**
Ubuntu: https://packages.ubuntu.com/bionic/libneon27-gnutls-dev
Debian: https://packages.debian.org/stretch/libneon27-gnutls-dev
Gentoo: https://packages.gentoo.org/packages/net-libs/gnutls

### CODE REVIEW

**Comment by tfoote** [comment]
Are you trying to add gnutls in general or neon27? From a quick read he Gentoo package you've added is more equivalent to the dependency https://packages.ubuntu.com/xenial/libgnutls-dev

**Comment by pietrocolombo** [comment]
I think this is right for Gentoo: https://packages.gentoo.org/packages/net-libs/neon

**Review by tfoote — APPROVED** [review]

File: rosdep/base.yaml [hunk 1]
```diff
@@ -2239,6 +2239,10 @@ libncurses-dev:
   gentoo: [sys-libs/ncurses]
   macports: [ncurses]
   ubuntu: [libncurses5-dev]
+libneon27-gnutls-dev:
+  debian: [libneon27-gnutls-dev]
+  gentoo: [neon]
```
Comment (mikaelarguedas) [inline rosdep/base.yaml]: The gentoo package should list the category: `net-libs/neon`

Comment (pietrocolombo) [reply rosdep/base.yaml]: ok I did

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks @pietrocolombo for iterating

**Pull request merged**